Create Draft- Executive vote - September 13, 2023.md (#2453)
makerdao/community: governance/votes/Executive vote - September 13, 2023.md

# [Executive Proposal] Stability Scope Parameter Changes, Spark Protocol D3M Parameter Changes, Set Fortunafi Debt Ceiling to Zero DAI, DAO Resolution for HV Bank, Delegate Compensation and Other Actions - September 13, 2023

The Reserve Governance Facilitators, Sidestream, Dewiz and Phoenix Labs have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following actions within the Maker Protocol.

If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.

---

## Executive Summary

If this executive proposal passes, the following actions will occur within the Maker Protocol:

- Several Stability Scope parameter changes will be executed, including changes to Stability Fees and activation of the DC-IAM for PSM-PAX-A. 
- Adjustments will be made to the Spark Protocol Dai Direct Deposit Module (D3M) Maximum Debt Ceiling and Ceiling Increase Cooldown. 
- The Fortunafi vault's Debt Ceiling will be set to zero DAI, per the Stability Scope Artifact's requirement to offboard legacy Legal Recourse Assets.

> - A DAO resolution pertaining to the HV Bank vault will be approved to terminate future purchases and return available cash from the HVB Master Participation Trust. This is per the Stability Scope Artifact's requirement to offboard legacy Legal Recourse Assets.

Reviewer: I would suggest removing the second sentence. It is not part of the executive summary of actions.

date: 2023-09-13T00:00:00.000Z
address: "$spell_address"

---

# [Executive Proposal] Scope-Defined Parameter Changes, Spark Protocol D3M Parameter Changes, Set Fortunafi Debt Ceiling to Zero, DAO Resolution for HV Bank, Delegate Compensation and Other Actions - September 13, 2023

The Reserve Governance Facilitators, Sidestream, Dewiz and Phoenix Labs have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following actions within the Maker Protocol.

If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.

---

## Executive Summary

If this executive proposal passes, the following actions will occur within the Maker Protocol:

- Several Stability Scope-defined parameter changes will be executed, including changes to Stability Fees and the PSM-USDP-A DC-IAM Maximum Debt Ceiling.
- The Spark Protocol Dai Direct Deposit Module (D3M) Maximum Debt Ceiling will be increased to 400 million DAI; and the Ceiling Increase Cooldown will be increased to 12 hours. 

> - The Fortunafi vault's Debt Ceiling will be set to zero DAI, per the Stability Scope Artifact's requirement to offboard legacy Legal Recourse Assets.

Reviewer 2: ```suggestion
- The Fortunafi vault's Debt Ceiling will be set to 0 DAI, per the Stability Scope Artifact's requirement to offboard legacy Legal Recourse Assets.
```

- A DAO resolution pertaining to the HV Bank vault will be approved to terminate future purchases and return available cash from the HVB Master Participation Trust. This is per the Stability Scope Artifact's requirement to offboard legacy Legal Recourse Assets.
- Aligned Delegate compensation for August 2023 will be distributed.

> - A Spark Protocol Proxy Spell will be triggered that implements changes to the DAI borrow spread and the flash loan fee.

Author: Valuable context, thank you.

**Voting for this executive proposal will place your MKR in support of the actions outlined above.**

Unless otherwise noted, the actions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the actions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.

If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.

---

## Proposal Details

For voters' convenience, each proposed item is prefaced with information showing its provenance. 

- "Authorization" links to the official document(s) or governance action authorizing the proposed item's inclusion in the Executive Vote. This can be the Atlas, a Scope Alignment Artifact, a successful governance poll and/or a ratified budget proposal. 
- "Reasoning" links to the MakerDAO Forum post providing the rationale and additional context for each item. 

> - "Authorizing Parties" may also be listed (with or without links). These may include the responsible Scope Facilitator and/or Scope Advisory Council members.

Reviewer 2: I really don't think this description is necessary. It just adds bloat. It's fairly obvious what each item is referencing

> ### Stability-Scope Defined Parameter Changes

Reviewer 2: Haven't checked values, JS/VW can sign off on them

*Authorization*: [Stability Scope Artifact](https://mips.makerdao.com/mips/details/MIP104#0-the-stability-scope)\
*Reasoning*: [Forum post](https://forum.makerdao.com/t/stability-scope-parameter-changes-5/21969/1) "Stability Scope Parameter Changes"\
*Authorizing Party:* [Stability Scope Facilitator](http://forum.makerdao.com/t/stability-scope-parameter-changes-5/21969/4)

> The Scope-defined parameter changes below have been proposed by Stability Scope Advisory Council Member [Block Analitica (BA Labs)](https://forum.makerdao.com/t/professional-ecosystem-actor-introduction-ba-labs/20813): 

Reviewer 2: why not just add a "*Proposal*:" section along with the other things you've selected above, will reduce word count

#### [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) Changes

- ETH-A Stability Fee will be increased by 0.12% from 3.58% to **3.70%**.
- ETH-B Stability Fee will be increased by 0.12% from 4.08% to **4.20%**.
- ETH-C Stability Fee will be increased by 0.12% from 3.33% to **3.45%**.

> #### PSM-USDP-A DC-IAM [Maximum Debt Ceiling](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) Change

Reviewer 2: vault name is PSM-PAX-A

> - Maximum Debt Ceiling will be increased from 0 (zero) to **120 million DAI**.

Reviewer 2: add delta and units

```suggestion
- Maximum Debt Ceiling will be increased by 120 million DAI from 0 DAI to **120 million DAI**.
```

> - Maximum Debt Ceiling will be increased from 0 (zero) to **120 million DAI**.

Reviewer 2: what are the other DC-IAM parameters going to be? gap and ttl should be listed here (i'm surprised the devs haven't asked for these values tbh)

> - Maximum Debt Ceiling will be increased from 0 (zero) to **120 million DAI**.

Reviewer 2: need to confirm and add other DC-IAM parameters.

DC-IAM is a collection of three parameters - line, gap, and ttl.

### Spark Protocol Dai Direct Deposit Module (D3M) Parameter Changes

*Authorization*: [Governance Poll 2023-09-04](https://vote.makerdao.com/polling/QmQnUhZt#poll-detail)\
*Reasoning*: [Forum post](http://forum.makerdao.com/t/upcoming-spell-proposed-changes/21801) "Upcoming Spell Proposed Changes"\

> *Authorizing Parties:* [Stability Scope Facilitator](http://forum.makerdao.com/t/upcoming-spell-proposed-changes/21801/4); [BA Labs](http://forum.makerdao.com/t/upcoming-spell-proposed-changes/21801/5)

Reviewer 2: The Authorizing party is governance voters who have approved the poll. As written it implies that the Scope Facilitator has pushed this to the exec level which is not the case. The Facilitator approved the poll creation. BA Labs do not have any governance mandate for authorizing these changes at either poll or exec level

> Phoenix Labs has proposed increasing the Spark Protocol D3M [Maximum Debt Ceiling](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) parameter as SparkLend's DAI borrow market is maxed out and requires capacity for more users. Phoenix Labs has also proposed increasing the D3M [Ceiling Increase Cooldown](https://manual.makerdao.com/module-index/module-dciam#ceiling-increase-cooldown-ttl) parameter to reduce exposure to Maker Core should a time-sensitive shutdown be required. If this executive proposal passes, the following actions will occur:

Reviewer 2: IMO this is unnecessary and adds bloat but is ultimately a style choice

> - The Spark Protocol D3M (DIRECT-SPARK-DAI) Maximum Debt Ceiling will be increased from 200 million DAI to **400 million DAI.**
> - The Spark Protocol D3M (DIRECT-SPARK-DAI) Ceiling Increase Cooldown will be increased from 8 hours to **12 hours.**

Reviewer 2: I like to include the delta because sometimes the coded value will be a delta, so it provides all the values necessary for voters when they are reviewing

```suggestion
- The Spark Protocol D3M (DIRECT-SPARK-DAI) Maximum Debt Ceiling will be increased by 200 million DAI from 200 million DAI to **400 million DAI.**
- The Spark Protocol D3M (DIRECT-SPARK-DAI) Ceiling Increase Cooldown will be increased by 4 hours from 8 hours to **12 hours.**
```

Reviewer 2: you've also included deltas in the scope-defined section so should aim for consistency

Author: Hi Patrick, I included the deltas for SF changes because these are more "customer facing" per LFW. :)

> ### Fortunafi (RWA005-A) Vault - Initiate Offboarding

Reviewer 2: authorizing party is governance

*Authorization*: [Stability Scope Artifact](https://mips.makerdao.com/mips/details/MIP104#14-4-legacy-legal-recourse-assets); [Governance Poll 2023-08-28](https://vote.makerdao.com/polling/Qmcb1c9x)\
*Reasoning*: [Forum post](http://forum.makerdao.com/t/request-to-poll-offboarding-legacy-legal-recourse-assets/21582) "Offboarding Legacy Legal Recourse Assets"\

> *Authorizing party*:  [Stability Scope Facilitator](http://forum.makerdao.com/t/request-to-poll-offboarding-legacy-legal-recourse-assets/21582/8)

Reviewer 2: see above authorizing party is governance IMO

> The Stability Scope Artifact requires that legacy Legal Recourse Assets be offboarded prior to the launch of NewChain. If this executive proposal passes, the Fortunafi vault Debt Ceiling will be decreased from 15 million DAI to **zero DAI**.

Reviewer 2: capitals, use numbers, add delta

```suggestion
The Stability Scope Artifact requires that Legacy Legal Recourse Assets be offboarded prior to the launch of NewChain. If this executive proposal passes, the Fortunafi vault Debt Ceiling will be decreased by 15 million DAI from 15 million DAI to **0 DAI**.
```

> ### DAO Resolution for HV Bank (RWA009-A) Vault

Reviewer 2: authorizing party is governance

*Authorization*: [Stability Scope Artifact](https://mips.makerdao.com/mips/details/MIP104#14-4-legacy-legal-recourse-assets); [Governance Poll 2023-08-28](https://vote.makerdao.com/polling/QmNgKzcG)\
*Reasoning*: [Forum post](http://forum.makerdao.com/t/request-to-poll-offboarding-legacy-legal-recourse-assets/21582) "Offboarding Legacy Legal Recourse Assets"\

> *Authorizing party*:  [Stability Scope Facilitator](http://forum.makerdao.com/t/request-to-poll-offboarding-legacy-legal-recourse-assets/21582/8)

Reviewer 2: see above, authorizing party is governance IMO

> The Stability Scope Artifact requires that legacy Legal Recourse Assets be offboarded prior to the launch of NewChain. The Stability Scope Facilitator seeks approval of a DAO Resolution that will instruct the RWA Foundation to terminate future purchases in HV Bank’s RWA Master Participation Trust and return excess available cash to the DAO. 

Reviewer 2: capital "L"

```suggestion
The Stability Scope Artifact requires that Legacy Legal Recourse Assets be offboarded prior to the launch of NewChain. The Stability Scope Facilitator seeks approval of a DAO Resolution that will instruct the RWA Foundation to terminate future purchases in HV Bank’s RWA Master Participation Trust and return excess available cash to the DAO. 
```

> The Stability Scope Artifact requires that legacy Legal Recourse Assets be offboarded prior to the launch of NewChain. The Stability Scope Facilitator seeks approval of a DAO Resolution that will instruct the RWA Foundation to terminate future purchases in HV Bank’s RWA Master Participation Trust and return excess available cash to the DAO. 

Reviewer 2: ```suggestion
The Stability Scope Artifact requires that Legacy Legal Recourse Assets be offboarded prior to the launch of NewChain. The Stability Scope Facilitator seeks approval of a DAO Resolution that will instruct the RWA Foundation to terminate future purchases in HV Bank’s RWA Master Participation Trust and return excess available cash to the DAO. 
```

If this executive proposal passes, the DAO Resolution with the IPFS hash [QmXU2TwsRpVevGY74NVFbD9bKwtsw1mSuSce7My1zinD9m](https://gateway.pinata.cloud/ipfs/QmXU2TwsRpVevGY74NVFbD9bKwtsw1mSuSce7My1zinD9m) will be approved.

> ### Aligned Delegate Compensation for August 2023

Reviewer 2: To be clear, I haven't checked amounts or addresses, JS/VW can sign off on this

| Cloaky      | [0x869b6d5d8FA7f4FFdaCA4D23FFE0735c5eD1F818](https://etherscan.io/address/0x869b6d5d8FA7f4FFdaCA4D23FFE0735c5eD1F818) | 7.17         |
| WBC         | [0xeBcE83e491947aDB1396Ee7E55d3c81414fB0D47](https://etherscan.io/address/0xebce83e491947adb1396ee7e55d3c81414fb0d47) | 6.72         |
| BLUE        | [0xb6c09680d822f162449cdfb8248a7d3fc26ec9bf](https://etherscan.io/address/0xb6c09680d822f162449cdfb8248a7d3fc26ec9bf) | 1.25         |

*Note*: Per the Governance Scope, there are only 10 paid Aligned Delegate slots. QGov and WBC received payments in August because they were previously ranked ADs who became unranked in September. Therefore, per [the Atlas](https://mips.makerdao.com/mips/details/MIP101#2-6-3-4-1), they were allocated payments from their respective AD Buffer. 

### Spark Protocol Proxy Spell

If this executive proposal passes, a [Proxy Spell](https://forum.makerdao.com/t/executive-proxies-and-spark-protocol/20976) for Spark Protocol implementing the following changes will be triggered at [0x95bcf659653d2E0b44851232d61F6F9d2e933fB1](https://etherscan.io/address/0x95bcf659653d2E0b44851232d61F6F9d2e933fB1):

> #### DAI Borrow Spread Change

Reviewer 2: authorizing party is governance

*Authorization*: [Governance Poll 2023-09-04](https://vote.makerdao.com/polling/QmQrkxud)\
*Reasoning*: [Forum Post](https://forum.makerdao.com/t/upcoming-spell-proposed-changes/21801) "Upcoming Spell Proposed Changes"\
*Authorizing Party*: [Stability Scope Facilitator](http://forum.makerdao.com/t/upcoming-spell-proposed-changes/21801/4)

Phoenix Labs has proposed increasing the DAI borrow spread; otherwise, there would be zero-risk in farming the [airdrop](https://forum.makerdao.com/t/sparkdao-spk-pre-farming-airdrop-general-subdao-farming-overview/21595). The Proxy Spell will increase the DAI borrow spread from 0% to **0.5%**.

> #### Flash Loan Fee Change

Reviewer 2: authorizing party is governance

*Authorization*: [Governance Poll 2023-09-04](https://vote.makerdao.com/polling/QmbCDKof#poll-detail)\
*Reasoning*: [Forum Post](https://forum.makerdao.com/t/upcoming-spell-proposed-changes/21801) "Upcoming Spell Proposed Changes"\
*Authorizing Party*: [Stability Scope Facilitator](http://forum.makerdao.com/t/upcoming-spell-proposed-changes/21801/4)

The Flash Loan Fee was reset to 0.09% in the pool upgrade and needs to be lowered again to its previous level. The Proxy Spell will reduce the [Flash Loan Fee](https://docs.sparkprotocol.io/developers/guides/flash-loans#flash-loan-fee) from 0.09% to **0%**.

### Core Unit MKR Vesting Transfers

#### DECO-001 Core Unit

*Authorization*: [Ratified MKR budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP36#sentence-summary)

> The [DECO-001](https://mips.makerdao.com/mips/details/MIP39c2SP23#mission) Core Unit manages the Deco fixed-rate protocol. The proposed action concerns the transfer of **125 MKR** to the Core Unit's Auditor Wallet ([0xF482D1031E5b172D42B2DAA1b6e5Cbf6519596f7](https://etherscan.io/address/0xF482D1031E5b172D42B2DAA1b6e5Cbf6519596f7)). If this executive proposal passes, the transfer will be executed.

Reviewer 2: You don't need to describe the role of the CU IMO, just adds word count without much benefit

#### SES-001 Core Unit

*Authorization*: [Ratified MKR budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP17#sentence-summary)

> The Sustainable Ecosystem Scaling (SES) Core Unit is the Responsible Facilitator for the Support, Protocol, Stability and Accessibility Scopes. The proposed action concerns the transfer of **34.94 MKR** to the Core Unit's Auditor Wallet ([0x87acdd9208f73bfc9207e1f6f0fde906bca95cc6](https://etherscan.io/address/0x87acdd9208f73bfc9207e1f6f0fde906bca95cc6)). If this executive proposal passes, the transfer will be executed.

Reviewer 2: see above

---
title: Template - [Executive Vote] Scope-Defined Parameter Changes, Spark Protocol D3M Parameter Changes, Set Fortunafi Debt Ceiling to Zero, DAO Resolution for HV Bank, Delegate Compensation and Other Actions - September 13, 2023

> summary: Scope-defined parameter changes; Spark Protocol D3M parameter changes; set Fortunafi Debt Ceiling to zero; approve DAO Resolution for HV Bank to terminate future purchases; Aligned Delegate compensation for August 2023; trigger Spark Protocol Proxy Spell; Core Unit MKR vesting transfers; scuttle MCD_CAT contract.  

Reviewer: The summary should contain more information than the title. Although it's a lengthy summary, it should still provide valuable details. Here are some suggestions:

- Specify which parameter fields are being modified.
- Combine information about Spark changes to encompass alterations to Spark D3M and spell execution.
- Remove "approve" from HBV.
- Include MKR expenditures.
- Add housekeeping items.

Author: The summary can be more a matter of personal preference, I think.

As for my summary here, I wrote this in accordance with how I did things during our training, and it also aligns with previous Exec vote examples. The summary lists all items, versus just the higher-impact ones, but I personally don't think the summary should be packed full of details.

Reviewer: The issue is that presently, the summary isn't providing much value as it merely repeats the title (often verbatim). It's not a critical problem, but it ties into my other feedback about adding unnecessary words and length to the spell without enhancing its utility for MKR voters.

date: 2023-09-13T00:00:00.000Z
address: "$spell_address"

> 

Reviewer: Eliminate unnecessary lines.

Author: done, thanks

---

# [Executive Proposal] Scope-Defined Parameter Changes, Spark Protocol D3M Parameter Changes, Set Fortunafi Debt Ceiling to Zero, DAO Resolution for HV Bank, Delegate Compensation and Other Actions - September 13, 2023

The Reserve Governance Facilitators, Sidestream, Dewiz and Phoenix Labs have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following actions within the Maker Protocol.

If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.

---

## Executive Summary

> If this executive proposal passes, the following actions will occur within the Maker Protocol:

Reviewer: I'll omit my comments about the order, but can we organize these into coherent groups to enhance readability? Perhaps something like this:

- Parameter Modifications
- Updates on RWA
- Expenses
- Administrative Updates

I believe that if we're avoiding the use of "changes" or "additions" as GovAlpha used to do, we should introduce another way to prevent the executive summary from becoming a lengthy text block.

Author: Thanks JS for the suggestion!

After thinking on it, I don't think grouping items into sections/classes is wise in the Executive Summary. 

Here's the pros:  I tried drafting a Exec Summary v2 to see visually how it looks. It offers a clearer signal to readers in terms of similarities between items, and also gives a brief takeaway about the essence of each proposed item.

<img width="1019" alt="Screenshot 2023-09-09 at 3 27 13 PM" src="https://github.com/makerdao/community/assets/139828768/80b354de-6ba6-4e26-9129-3ad200ee8f2b">

But the good is outweighed I think by the cons. In our training, we were advised to "Order the items by judged impact/importance."

Items that are placed together into a single class do not necessarily have the same overall impact/importance in relation to the rest of the remaining items in the proposal. For instance, the 2 Spark Protocol-related Actions, which are grouped together in the Exec Summary v2 do not have the same judged impact/importance vis a vis the rest of the items. 

See this previous [Exec Vote](https://github.com/makerdao/community/blob/ca6fe657cccd887ac12a0ec15f80d412efa28b06/governance/votes/Executive%20Vote%20-%20August%202%2C%202023.md), where the Spark D3M parameter adjustments were listed much higher in the Copy than the Spark Proxy Spell, due to difference in impact. And in the immediate instance, I think the Spark Protocol parameter changes are more minor in nature. 

Another example:  2 RWA vault updates that are grouped together in the Exec Summary simply bc they share the same nature may be very different in terms of judged impact/importance. For example, one such update may be a complete restructuring of the vault (New Silver). While another vault update may involve a simple DAO resolution or a doc parameter update.

So if we went with the v.2 of Exec Summary, we would have 2 choices:

1.  change ordering of items in the **Proposal Details** section, so that the ordering matches that of the Executive Summary. We would thus break the principle cited above of ordering items by judged impact.

2.  On the other hand, we can choose to order items in **Proposal Details** section so as to adhere to the principle cited above. But in this case, the ordering of items in the above **Executive Summary** would not match the ordering of items in the below **Proposal Details** section. This would be v confusing to readers, I think, as the Executive Summary is meant to be a "table of contents" that is legible and convenient. 

I think it's best for the executive summary bullet points to match the order of the items in the below Proposal Details section. 

I agree that readability is key; and I think we have made significant improvements to breaking up the blocks of text by introducing the provenance preface for each item

As long as we keep the sentences in the Executive Summary relatively succinct, the Executive Summary will not be prohibitive in appearance/readability. 

In sum, I think in this instance, we should keep to current practice, especially for this first Exec Vote. 

@VoteWizard feel free to chime in with your thoughts as well. We can reconsider and implement at another date.

Reviewer 3: Thank you for the comprehensive discussion. I've read both perspectives and find merit in them.

However, I tend to agree with the guidelines we've previously adopted as per GovAlpha's practices. The principle of ordering items by their "judged impact/importance" makes a lot of sense and helps to direct attention to the most critical elements of a proposal.

The idea of grouping items into coherent categories like "Parameter Modifications," "Updates on RWA," "Expenses," and "Administrative Updates" has its merits in terms of readability, but as @Le-Bateleur-2023  pointed out, this could lead to a disconnect between the Executive Summary and the Proposal Details in terms of their order.

- Several Stability Scope-defined parameter changes will be executed, as detailed below.
- The Spark Protocol Dai Direct Deposit Module (D3M) Maximum Debt Ceiling will be increased to 400 million DAI; and the Ceiling Increase Cooldown will be increased to 12 hours. 
- The Fortunafi vault's Debt Ceiling will be set to zero DAI, per the Stability Scope Artifact's requirement to offboard legacy Legal Recourse Assets.
- A DAO resolution pertaining to the HV Bank vault will be approved to terminate future purchases and return available cash from the HVB Master Participation Trust. This is per the Stability Scope Artifact's requirement to offboard legacy Legal Recourse Assets.
- Aligned Delegate compensation for August 2023 will be distributed.

> - A Spark Protocol Proxy Spell will be triggered that implements changes to the DAI borrow spread and the flash loan fee.

Reviewer: I would add, however, that the two Spark actions should be grouped/adjacent for clarity.

- MKR vesting transfers will be executed for the DECO-001 and SES-001 Core Units, per their successful budget proposals. 
- The `MCD_CAT` contract will be scuttled by removing it from the Chainlog and removing all its permissions on the Protocol contracts.  

**Voting for this executive proposal will place your MKR in support of the actions outlined above.**

> Unless otherwise noted, the actions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the actions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.

Reviewer 3: Is the "DAO Resolution for HV Bank (RWA009-A) Vault" subjected to the GSM Pause?

Author: Yes, I am pretty certain it is. 

(I will ask Patrick to make sure about this as well.)

Reviewer 2: if you read the RWA foundation docs they take the resolution from the point of execution of the spell which is after the GSM Delay. The resolution itself also specifies it is effective as of "execution" of the vote

If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.

---

## Proposal Details

For voters' convenience, each proposed item is prefaced with information showing its provenance. 

> - "Authorization" links to the official document(s) or governance action authorizing the proposed item's inclusion in the Executive Vote. This can be the Atlas, a Scope Alignment Artifact, a governance poll and/or a ratified budget proposal. 

Reviewer: Remove examples.

Author: I'm leaving these examples in JS (as explained in earlier comment); as I believe the examples are not obvious/evident to non-insiders to Maker governance; and more of new/new-ish folks will certainly be entering the ecosystem in the near future. :)

- "Reasoning" links to the MakerDAO Forum post providing the rationale and additional context for each item. 
- "Authorizing Parties" may also be listed (with or without links). These may include the responsible Scope Facilitator and/or Scope Advisory Council members.

> ### Stability-Scope Defined Parameter Changes

Reviewer: Missing information from this section:

https://forum.makerdao.com/t/stability-scope-parameter-changes-5/21969#stability-scope-bounded-mutable-alignment-artifact-changes-8

Author: These are not included in the spell JS

Reviewer 2: Make sure the Stability Scope text has been updated to match the new values, although agree they are not needed in the spell

*Authorization*: [Stability Scope Artifact](https://mips.makerdao.com/mips/details/MIP104#0-the-stability-scope)\
*Reasoning*: [Forum post](https://forum.makerdao.com/t/stability-scope-parameter-changes-5/21969/1) "Stability Scope Parameter Changes"\
*Authorizing Party:* [Stability Scope Facilitator](http://forum.makerdao.com/t/stability-scope-parameter-changes-5/21969/4)

The Scope-defined parameter changes below have been proposed by Stability Scope Advisory Council Member [Block Analitica (BA Labs)](https://forum.makerdao.com/t/professional-ecosystem-actor-introduction-ba-labs/20813): 

#### [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) Changes

- ETH-A Stability Fee will be increased by 0.12% from 3.58% to **3.70%**.
- ETH-B Stability Fee will be increased by 0.12% from 4.08% to **4.20%**.
- ETH-C Stability Fee will be increased by 0.12% from 3.33% to **3.45%**.

#### PSM-USDP-A DC-IAM [Maximum Debt Ceiling](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) Change

- Maximum Debt Ceiling will be increased from 0 (zero) to **120 million DAI**.

> ### Spark Protocol Dai Direct Deposit Module (D3M) Parameter Changes

Reviewer: What happened to the flash loan fee and borrow spread increase?

Author: Further below, as they are distinct items with differing impact/importance.

> *Authorization*: [Governance Poll 2023-09-04](https://vote.makerdao.com/polling/QmQnUhZt#poll-detail)\

Reviewer 3: One small point I'd like to bring up is regarding the term "Governance Poll 2023-09-04." To add clarity and accuracy, might we consider phrasing it as "Successful outcome of Governance Poll 2023-09-04"?
See LFW's comment for reasoning:
https://github.com/makerdao/community/pull/2394/files#r1293471577

Author: Thanks VW! LFW suggested similar to me, but opined it may be unnecessary. I do think it's redundant/unnecessary in the preface before each item. An "unsuccessful" poll would never reasonably be used as an authorization for a Governance action. It's assumed that the Poll is linked because its outcome supports the inclusion. 

I can revise the introductory section in the Proposal Details, to stress that a successful Poll is one of the many authorization sources that can be listed. (I also think this is redundant, but we can cover all of our bases with this addition.)

### DAO Resolution for HV Bank (RWA009-A) Vault

*Authorization*: [Stability Scope Artifact](https://mips.makerdao.com/mips/details/MIP104#14-4-legacy-legal-recourse-assets); [Governance Poll 2023-08-28](https://vote.makerdao.com/polling/QmNgKzcG)\
*Reasoning*: [Forum post](http://forum.makerdao.com/t/request-to-poll-offboarding-legacy-legal-recourse-assets/21582) "Offboarding Legacy Legal Recourse Assets"\
*Authorizing party*:  [Stability Scope Facilitator](http://forum.makerdao.com/t/request-to-poll-offboarding-legacy-legal-recourse-assets/21582/8)

The Stability Scope Artifact requires that legacy Legal Recourse Assets be offboarded prior to the launch of NewChain. The Stability Scope Facilitator seeks approval of a DAO Resolution that will instruct the RWA Foundation to terminate future purchases in HV Bank’s RWA Master Participation Trust and return excess available cash to the DAO. 

If this executive proposal passes, the DAO Resolution with the IPFS hash [QmXU2TwsRpVevGY74NVFbD9bKwtsw1mSuSce7My1zinD9m](https://gateway.pinata.cloud/ipfs/QmXU2TwsRpVevGY74NVFbD9bKwtsw1mSuSce7My1zinD9m) will be approved.

> For reference, a raw version of the text may be found below:

Reviewer 3: Given that you've already included a link to the DAO resolution, the raw text might be redundant and could make the document harder to digest. Could we consider keeping just the link?

Author: Good point. If we can make the Exec more brief without losing any vital info, we should. I'll remove it then. Thanks VW.

| Cloaky      | [0x869b6d5d8FA7f4FFdaCA4D23FFE0735c5eD1F818](https://etherscan.io/address/0x869b6d5d8FA7f4FFdaCA4D23FFE0735c5eD1F818) | 7.17         |
| WBC         | [0xeBcE83e491947aDB1396Ee7E55d3c81414fB0D47](https://etherscan.io/address/0xebce83e491947adb1396ee7e55d3c81414fb0d47) | 6.72         |
| BLUE        | [0xb6c09680d822f162449cdfb8248a7d3fc26ec9bf](https://etherscan.io/address/0xb6c09680d822f162449cdfb8248a7d3fc26ec9bf) | 1.25         |

*Note*: Per the Governance Scope, there are only 10 paid Aligned Delegate slots. QGov and WBC received payments in August because they were previously ranked ADs who became unranked in September. Therefore, per [the Atlas](https://mips.makerdao.com/mips/details/MIP101#2-6-3-4-1), they were allocated payments from their respective AD Buffer. 

### Spark Protocol Proxy Spell

A [Proxy Spell](https://forum.makerdao.com/t/executive-proxies-and-spark-protocol/20976) for Spark Protocol at [0x95bcf659653d2E0b44851232d61F6F9d2e933fB1](https://etherscan.io/address/0x95bcf659653d2E0b44851232d61F6F9d2e933fB1) will be triggered if this executive proposal passes.

> 

Reviewer 3: ```suggestion
This proxy spell contains the following changes:

```

Author: I'll incorporate this suggestion but use a different phrasing to emphasize the conditional (if/then) nature. Thanks VW!

*Authorizing Party*: [Stability Scope Facilitator](http://forum.makerdao.com/t/upcoming-spell-proposed-changes/21801/4)

Phoenix Labs has proposed increasing the DAI borrow spread; otherwise, there would be zero-risk in farming the [airdrop](https://forum.makerdao.com/t/sparkdao-spk-pre-farming-airdrop-general-subdao-farming-overview/21595). The Proxy Spell will increase the DAI borrow spread from 0% to **0.5%**.

#### Flash Loan Fee Change

*Authorization*: [Governance Poll 2023-09-04](https://vote.makerdao.com/polling/QmbCDKof#poll-detail)\
*Reasoning*: [Forum Post](https://forum.makerdao.com/t/upcoming-spell-proposed-changes/21801) "Upcoming Spell Proposed Changes"\
*Authorizing Party*: [Stability Scope Facilitator](http://forum.makerdao.com/t/upcoming-spell-proposed-changes/21801/4)

The Flash Loan Fee was reset to 0.09% in the pool upgrade and needs to be lowered again to its previous level. The Proxy Spell will reduce the [Flash Loan Fee](https://docs.sparkprotocol.io/developers/guides/flash-loans#flash-loan-fee) from 0.09% to **0%**.

### Core Unit MKR Vesting Transfers

> #### A) DECO-001 Core Unit

Author: Will remove, thanks JS

The [DECO-001](https://mips.makerdao.com/mips/details/MIP39c2SP23#mission) Core Unit manages the Deco fixed-rate protocol. The proposed action concerns the transfer of **125 MKR** to the Core Unit's Auditor Wallet ([0xF482D1031E5b172D42B2DAA1b6e5Cbf6519596f7](https://etherscan.io/address/0xF482D1031E5b172D42B2DAA1b6e5Cbf6519596f7)). If this executive proposal passes, the transfer will be executed.

#### B) SES-001 Core Unit

*Authorization*: [Ratified MKR budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP17#sentence-summary)

The Sustainable Ecosystem Scaling (SES) Core Unit is the Responsible Facilitator for the Support, Protocol, Stability and Accessibility Scopes. The proposed action concerns the transfer of **34.94 MKR** to the Core Unit's Auditor Wallet ([0x87acdd9208f73bfc9207e1f6f0fde906bca95cc6](https://etherscan.io/address/0x87acdd9208f73bfc9207e1f6f0fde906bca95cc6)). If this executive proposal passes, the transfer will be executed.

### Scuttle MCD_CAT Contract

*Reasoning*: Forum Post "[Proposal to Scuttle MCD_CAT](https://forum.makerdao.com/t/proposal-to-scuttle-mcd-cat-upcoming-executive-spell-2023-09-13/21958)"\

> *Authorizing Party*: Protocol Scope Facilitators {**LINK MISSING**}

Reviewer: Noting the missing link, must be added before copy is approved.

Reviewer 2: @Le-Bateleur-2023 don't resolve comments that aren't resolved. you will miss things doing this
